### PR TITLE
[#1009] Trigger status update on edit

### PR DESCRIPTION
**Pull request by mtwestra**
* Updating the status of a cascade resource is
triggered manually.

### CODE REVIEW

**Comment by iperdomo** [comment]
## Checklist

* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting

File: Dashboard/app/js/lib/controllers/data-controllers.js [hunk 1]
```diff
@@ -106,6 +107,10 @@ FLOW.cascadeResourceControl = Ember.ArrayController.create({
 		FLOW.store.findQuery(FLOW.Question, {cascadeResourceId: FLOW.selectedControl.selectedCascadeResource.get('keyId')});
 	}.observes('FLOW.selectedControl.selectedCascadeResource'),
 
+	triggerStatusUpdate: function(){
+		this.set('statusUpdateTrigger',!this.get('statusUpdateTrigger'));
```
Comment (iperdomo) [inline Dashboard/app/js/lib/controllers/data-controllers.js]: There is a built-in feature for this: `this.toggleProperty('statusUpdateTrigger')` do you think is worth changing?

Comment (mtwestra) [inline Dashboard/app/js/lib/controllers/data-controllers.js]: yes, let me change that. So I will remember it next time :-)

**Comment by iperdomo** [comment]
:+1:

**Pull request merged**